portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -82,4 +91,21 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
+
+  /** Returns the integer entered by user on how many comments  */
+  private int getMaxComments(HttpServletRequest request) {
+    // Get the input from the form on MaxComments and convert to integer to the
+    // toGet function
+    String CommentNumString = request.getParameter("max-comments");
+    // Convert the input to an int.
+    int CommentNum;
+    try {
+      CommentNum = Integer.parseInt(CommentNumString);
```

Comment:
Is the string allowed to be a float? I'm wondering if the form can be restricted to only accept ints and not floats, etc.

Reply:
The form on the index.html only accepts integers